wp-content/themes/haruoshi_origin/single-works.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * This is single-works.php
+ *
+ * @package haruoshi-origin
+ */
+
+get_header(); ?>
+
+<article class="container">
+	<h1 class="container__title"><?php the_title(); ?></h1>
+	<p class="container__date"><?php the_time( 'Y/m/d' ); ?></p>
+	<?php if ( has_post_thumbnail() ): ?>
+		<img class="container__thumbnail" src="<?= the_post_thumbnail_url(); ?>" alt="thumbnail">
+	<?php else: ?>
+		<img class="container__thumbnail" src="<?= esc_url( get_stylesheet_directory_uri() . '/img/dummy.jpg' ); ?>" alt="dummy-thumbnail">
+	<?php endif; ?>
+	<p class="container__contents"><?php the_field( 'contents' ); ?></p>
```

Comment:
```suggestion
	<p class="container__contents"><?php /* ここが違う気がする */ ?></p>
```
本文はカスタムフィールドじゃないからこれだと本文表示できない気がする

Reply:
一応表示されている
あと本文はカスタムフィールドとして作ってある気がする
![スクリーンショット 2022-03-20 19 53 39](https://user-images.githubusercontent.com/21308920/159158894-fb2bcf3d-0208-45e0-87af-06a1ee77d77f.png)
![スクリーンショット 2022-03-20 19 54 42](https://user-images.githubusercontent.com/21308920/159158951-20c486bf-2901-4cfb-9b28-3f20d9be1127.png)

Reply:
本文はカスタムフィールドじゃなくてデフォルトのエディターに入力したテキストを表示するようにお願いします

Reply:
done https://github.com/oooshima/origin_wordpress/pull/28/commits/61d6f1140404f07e63c2a12b08ee14eeae97c179